epochStart/bootstrap/disabled/incomingHeaderSubscriber.go

```diff
@@ -0,0 +1,25 @@
+package disabled
+
+import (
+	"github.com/multiversx/mx-chain-core-go/data"
+	"github.com/multiversx/mx-chain-core-go/data/sovereign"
+)
+
+// IncomingHeaderSubscriber -
+type IncomingHeaderSubscriber struct {
+}
+
+// AddHeader does nothing
+func (ihs *IncomingHeaderSubscriber) AddHeader(_ []byte, _ sovereign.IncomingHeaderHandler) error {
+	return nil
+}
+
+// CreateExtendedHeader does nothing
+func (ihs *IncomingHeaderSubscriber) CreateExtendedHeader(_ sovereign.IncomingHeaderHandler) (data.ShardHeaderExtendedHandler, error) {
+	return nil, nil
```

Comment:
nil nil return is not ok.

You need to return an empty structure here and nil for error.

Reply:
Right, good pointer, returned empty struct